cdap-ui/app/features/tracker/templates/entity.html

```diff
@@ -48,6 +48,10 @@
       <li ng-class="{'active': $state.is('tracker.detail.entity.audit')}">
         <a ui-sref="tracker.detail.entity.audit">Audit Log</a>
       </li>
+      <li ng-class="{'active': $state.is('tracker.detail.entity.preview')}"
+          ng-if="$state.params.entityType === 'streams' || EntityController.datasetExplorable">
```

Comment:
If this is one time lets try single binding, `::($state.params.entityType === 'streams' || EntityController.datasetExplorable)`